### ISSUE TITLE
BarChart secondaryData is completely ignored

### ISSUE THREAD

**Issue by ilyabreev** [Abhinandan-Kushwaha/react-native-gifted-charts#549]
Trying to do a bar chart with two y axis with different scale. But `secondaryData` property is ignored. All data is set with `data` like this: 

```
barData.push({
  value: count * 1000,
  frontColor: '#125BFF',
  spacing: 0,
});
barData.push({
  value: miles,
  frontColor: '#FF9500',
});
```

What I'm trying to do is add blue bars to `data`, but orange bars to `secondaryData`, because they are different scaled values.

`maxValue` and `stepValue` are set different for main yAxis and secondaryYAxis

(Largest orange bar value is 8192 on the image below. It is showing up like its scaled for main y axis, but not for the secondary one)

<img width="340" alt="Снимок экрана 2024-02-20 в 16 33 08" src="https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/assets/1045399/b06e6018-390a-4551-84b6-b9002585126f">

**Comment by gloriamannok** [comment]
Same here, any updates? 

I'm trying to have bars for individual values (primary data) and a line for accumulative values (secondary data). 
The line is not following secondary Y-axis scale :( 

```
const data = [200,125,175,50], 
  accData = [200,325,500,550];

...

<BarChart
  frontColor={greenColor}
  data={data}
  maxValue={600}
  showLine
  lineConfig={{ /* for hiding the line for primary data */
      startIndex: 0,
      endIndex: 0,
      hideDataPoints: true,
      isSecondary: false
  }}
  lineConfig2={{
      color: blueColor,
      thickness: 3,
      hideDataPoints: true,
      isSecondary: true
  }}
  lineData2={accData}
  secondaryYAxis={{
      maxValue: accData[accData.length-1].value + 10
  }}
/>
```
![IMG_5687](https://github.com/user-attachments/assets/ebc75e12-9f00-4b71-8f78-ec70c5eddf22)

**Comment by Abhinandan-Kushwaha** [comment]
Hi @ilyabreev 👋
Secondary Bars can now be achieved by setting `isSecondary: true` inside the data array objects. See https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/discussions/828

I have removed the `secondaryData` prop from Bar charts, as it was useless.

This is available from versions `1.4.35` onwards. Please use the latest version of the library.

@gloriamannok 👋
Can you try again in the latest version of the library? I have made fixes and refactors related to secondary data and the secondary Y-axis.
Also, you don't have to use the hack using 2 lines and 2 lineConfigs. Just have one `lineConfig`, and inside this set `isSecondary: true`. It will show the line from the `lineData` prop. If the `lineData` prop is not provided, then it renders the line from the `data` prop.
